Update roadmap for Q1 (#25555)
containers/podman: ROADMAP.md

one or more engineers.

## Future feature considerations

> The following features are of general important to Podman.  While these features have no timeline

Reviewer: ```suggestion
The following features are of general importance to Podman.  While these features have no timeline
```

associated with them yet, they will likely be on future quarterly milestones.

* Further improvements to `podman machine` to better support Podman Desktop and other developer usecases.
  - Smoother upgrade process for Podman machine operating system (OS) images
  - Convergence of WSL technologies with other providers including its OS

> * Remote client remote of OCI artifacts and its RESTFUL API

Reviewer 2: Maybe "for OCI artifacts"

* Integration of composefs

> * Partial pull support (zstd:chunked)

Reviewer 2: Isn't this done?

* Improved support for the BuildKit API.
* Performance and stability improvements.
* Reductions to the size of the Podman binary.

## Milestones and commitments by quarter

> This section is a historical account of what features were prioritized by quarter.  Results of the prioritization will be added before at the beginning of each quarter (Jan, Apr, July, Oct).

Reviewer: ```suggestion
This section is a historical account of what features were prioritized by quarter.  Results of the prioritization will be added prior to the start of each quarter (Jan, Apr, July, Oct).
```

### 2025 Q1 ####

#### Releases ####
- [x] Release Podman 5.4
- [x] Podman release automation

#### Features ####

> - [x] Artifact add --append

Reviewer: Is there any value in providing links to items when they are completed?

Author: hi @jakecorrenti for historical purposes, someone might find a use for it.  And I contemplated just doing it.  But it is a duplication (if not double duplication) and we always have git log and git for this sort of thing ... which in fact is the source of truth.  So I was kind of thinking of a wait and see on this one ... also, when we implement features, we dont always do so in a single PR and now i'm not sure what to do with it.  We could also link JIRA cards, but that can change in in a heartbeat between deletion, duplication, or archival.

Reviewer: All good points. I didn't have a preference either way, I was just curious.
